SCOPES.md

```diff
@@ -0,0 +1,70 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:signing`
+  * Used for Firefox/Fennec/Devedition style signing tasks
+
+* `project:comm:thunderbird:releng:signing`
+  * Used for Thunderbird style balrog tasks
+
```

Comment:
I forgot to say, this prefix is also supported: `project:mobile:focus:releng:signing:`

https://tools.taskcluster.net/groups/dypn8EpGTNOvC6egffix6Q/tasks/cRSdQzL7SoOofNdTiJTqsw/details